source/spark_sql_migrations/README.md

```diff
@@ -0,0 +1,67 @@
+# Spark SQL Migrations
+
+## Usage
+
+```python
+
+import spark_sql_migrations.migrations.schema_migration_pipeline as schema_migration_pipeline
+from spark_sql_migrations.container import create_and_configure_container
+from spark_sql_migrations.models.spark_sql_migrations_configuration import (
+    SparkSqlMigrationsConfiguration,
+)
+from spark_sql_migrations.models.table import Table
+from spark_sql_migrations.models.schema import Schema
+
+from pyspark.sql.types import (
+    StructType,
+    StructField,
+    StringType,
+)
+
+def migrate() -> None:
+    _configure_spark_sql_migration()
+    _migrate()
+
+
+def _configure_spark_sql_migration() -> None:
+    schema = StructType(
+        [
+            StructField("column1", StringType(), False),
+            StructField("column2", StringType(), False)
+        ]
+    )
+
+    schema_config = [
+        Schema(
+            name="test_schema",
+            tables=[
+                Table(name="test_table", schema=schema),
+                Table(name="test_table_2", schema=schema)
+            ]
+        )
+    ]
```

Comment:
What are these (schema and schema_config) used for?

Reply:
schema is being used in the schema_config. 
schema_config is used in the SparkSqlMigrationsConfiguration. The purpose is to tell the package what all our schemas look like and the names of the Schemas and Tables. 

The package needs this knowledge to check if the schemas and tables exist etc.